public/components/ctas/8.html

```diff
@@ -0,0 +1,26 @@
+<aside class="relative overflow-hidden text-gray-300 bg-gray-900 lg:flex justify-center">
+    <div class="w-full p-12 text-center lg:w-2/3 sm:p-16 lg:p-24">
+      <div class="max-w-xl mx-auto lg:ml-0">
+
+        <div class="p-2 border inline-block">community</div>
+
+        <p class="mt-4 text-2xl font-bold text-white sm:text-3xl w-2/3 text-center mx-auto">
+          Lorem, ipsum dolor sit amet consectetur.
+        </p>
+  
+        <p class="hidden lg:mt-4 lg:block mx-auto">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Et, egestas
+          tempus tellus etiam sed. Quam a scelerisque amet ullamcorper eu enim et
+          fermentum,
+        </p>
+
+        <a
```

Comment:
Could the `href=""` be replaced with `href="#"`?

Should `text-black` be `text-gray-900` so it pairs up nicely with the background color?

How does `hover:bg-yellow-400` look? And can we add `transition` to the classes for a smooth hover effect.

Reply:
Probably want to consider `focus:` styling as well.